tests/TestCase/Validation/ValidatorTest.php

```diff
@@ -1300,28 +1300,6 @@ public function testAllowEmptyDateTimeCondition(): void
         $this->assertEmpty($validator->validate($data, false));
     }
 
-    /**
-     * test allowEmptyDateTime with deprecated argument order
-     */
-    public function testAllowEmptyDateTimeDeprecated(): void
-    {
-        $validator = new Validator();
-        $this->deprecated(function () use ($validator): void {
-            $validator->allowEmptyDateTime('published', 'datetime required', 'update');
-        });
```

Comment:
Is this allowed now?

Reply:
see above testCase , it only is duplicated TestCase but without `deprecated()` right now
`$when` as second Argument allowed (with deprecation) in cake 3 (not sure after which version)
but now in cake 4 `$when` as second Argument in not allowed anymore